css/typography.css

```diff
@@ -0,0 +1,24 @@
+h1, .xlg-para {
+    font-size: 48px;
+    line-height: 64px;
+}
```

Comment:
I would suggest changing the class name to something easier to understand, maybe `2xl-para` or alternatively suggest `text-xl`, `text-lg`, etc as the naming style.